misc/PFS_Abbreviations.tsv

```diff
@@ -38,86 +39,91 @@ DRP	Software	General	Data Reduction Pipeline	Software for data processing. There
+fps	ICS	Actor	Fiber Positioning Sequencer	PFI software to operate fiber configuration. FPS controls MCS exposure, derives fibers positions from measured centroids, and sends them with command to MPS to move fibers. ASIAA has a responsibility for its development.
 FPSA	SpS	Subcomponent	Fiber Slit Positioning Sub-Assembly	SpS assembly to position fiber slit. FPSA consists of Hexapod to tilt and shift the Fiber Slit sub-Assembly (FSA).
-FSA	SpS/FOCCoS	Subcomponent	Fiber Slit Assembly	FOCCos assembly as a part of SpS. The slit accommodates about 600 fibers in a row.
+FSA	SpS/FOCCoS	Subcomponent	Fiber Slit Assembly	Focus assembly as a part of SpS. The slit accommodates about 600 fibers in a row.
 FST	SpS	Subcomponent	Fiber Slit alignment Tool	SpS integration tool to align Fiber Slit Assembly (FSA). Dedicated collimator corrector and camera are used.
 FUA	PFI	Subcomponent	Fixed Upper Assembly	One of the two main PFI structure. FUA is fixed to POpt2 and accommodates cable wrapper assembly,(CWA), modular plate for fiber cables (MP3), and calibration lamp and so on.
 fvc	ICS	Actor	Fiber Viewing Camera	PFI software to take images of fixed fiducial fibers. fvc also turn on/off LED to illuminate fixed fiducial fibers. ASIAA has a responsibility for its development.
 Gen2	Subaru	Operation	Generation-2 observation control system	Current observation control system (OCS) for Subaru. It launched in 2011.
 HSC	Subaru	Instrument	Hyper Suprime-Cam	Wide field (~1.5 degree in diameter) camera with more than 100 CCDs at Subaru prime focus. HSC SSP, which started in 2014, is sister survey of PFS in the SuMIRe project.
 ICD	General	Document	Interface Control Document	If there is an interface between different developer institutes, they discuss it using ICD.
 ICS	Software	General	Instrument Control Software	Software packages to operate PFS instrument. Individual actuators, which control smaller components, exchange command, status with each other via messaging hub systems (MHS).
-IIS	SpS	Subcomponent	Internal Illumination Source	Light source ofSpS to illuminate engineering fibers.
-IPMU	General	Organization	Kavli Institute for the Physics and Mathematics of the Universe, the University of Tokyo	PFS collaborator (PI) taking initiative to proceed the project. IPMU also has a responsibility to develop softwares interfacing between PFS and Subaru, and execute system commissioning.
-IR3	Subaru	Facility	IR-side 3rd floor in the dome building 	The place on which the UPS, chillers for spectrgraph modules and the spectrograph clean room (SCR) will be accomodated.
-IR4	Subaru	Facility	IR-side 4th floor in the dome building 	The place on which the spectrgraph clean room (SCR) will be constructed.
+iic	ICS	Actor	Instrument Interface and Controller	ICS software module which organizes operation of PFS subsystems in an exposure sequence, in harmony with the telescope.
```

Comment:
iic in lower or upper?

Reply:
Well... I don't know. but other actors are written in lower case.

Reply:
I think IIC as an acronym should be in upper case.

Reply:
Looking at the other actor abbreviations, perhaps it's better to leave it lower for consistency.